pkg/process/config/config.go

```diff
@@ -656,3 +686,37 @@ func setupLogger(loggerName config.LoggerName, logFile string, cfg *AgentConfig)
 		config.Datadog.GetBool("log_format_json"),
 	)
 }
+
+func getDDAgentClient() (pb.AgentClient, error) {
+	// This is needed as the server hangs when using "grpc.WithInsecure()"
+	tlsConf := tls.Config{InsecureSkipVerify: true}
```

Comment:
hmm, I can't comment on this but someone with more knowledge than myself might want to pipe in if this is ok to do in this context

Reply:
@truthbk - recommended this.